inc/classes/busting/class-facebook-sdk.php

```diff
@@ -455,6 +461,66 @@ private function get_locale_from_url( $url ) {
 		return $matches['locale'];
 	}
 
+	/**
+	 * Extract XFBML from a URL to bust.
+	 *
+	 * @since  3.4.3
+	 * @access private
+	 * @author Soponar Cristina
+	 *
+	 * @param  string $url Any string containing the URL to bust.
+	 * @return string|bool The XFBML on success. False on failure.
+	 */
+	private function get_xfbml_from_url( $url ) {
+		$pattern = '@//connect\.facebook\.net/(?<locale>[a-zA-Z_-]+)/sdk\.js#xfbml=(?<xfbml>[0-9]+)&version=(?<version>[a-zA-Z0-9.]+)&appId=(?<appId>[0-9]+)@i';
```

Comment:
This will fail if `xfbml`, `version`, and `appId` are not in that order.
```suggestion
		$pattern = '@//connect\.facebook\.net/(?<locale>[a-zA-Z_-]+)/sdk\.js#(?:.+&)?xfbml=(?<xfbml>[0-9]+)@i';
```